simulation-runner/run_sbt.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
Man, so many similar files.

I think we should get rid of this one and just inline the call to `sbt run` in the `on(gatling, ...)` call in `20_run_sbt.rb`.